ui/src/pages/faq.tsx

```diff
@@ -18,7 +18,7 @@ const faqContents = [
         id: 'what_octopus',
         heading: 'What is Octopus?',
         content:
-            "<p className='mb-2'>Octopus sets out to be the new primary research record: a place where researchers from all disciplines can freely record all of their efforts and findings without external requirements unduly shaping their outputs.</p><p className='mb-2'>This complements the traditional paper, which continues to be a good place to disseminate research summaries (and particularly ‘findings’) to a wider audience: publishing in Octopus doesn't stop you publishing in a journal down the line should you want – but it is the place where you establish your priority for all to see, and document your work in full detail in order to do so, rather like a patent office.</p> <p className='mb-2'>Our aim is that the different structure of Octopus will help researchers approach their work differently and allow people to concentrate on excellence rather than chasing ‘impact’.</p> "
+            "<p className='mb-2'>Octopus sets out to be the new primary research record: a place where researchers from all disciplines can freely record all of their research outputs without external requirements unduly shaping their outputs.</p><p className='mb-2'>This complements the traditional paper, which continues to be a good place to disseminate research summaries (and particularly &apos;findings&apos;) to a wider audience: publishing in Octopus doesn't stop you publishing in a journal down the line should you want – but it is the place where you establish your priority for all to see, and document your work in full detail in order to do so, rather like a patent office. Octopus publications are smaller units. These publications are linked together, reflecting different stages of the research process, allowing each piece of work to be done by different researchers at different times where appropriate. They can be published individually, without needing to fit it into the structure of a traditional journal article. To read more about Octopus' publication types, click <a className='underline' href='https://www.octopus.ac/faq#how_octopus' target='_blank'>here.</a></p><p className='mb-2'>You can also publish on Octopus any work you have previous published in a journal if you retained the copyright, or published CC-BY. Doing this allows your work to get higher visibility and can help it be recognised by your institution or funders.</p> <p className='mb-2'>Our aim is that the different structure of Octopus will help researchers approach their work differently and allow people to concentrate on intrinsic excellence rather than chasing &apos;impact&apos;.</p>"
```

Comment:
```suggestion
            "<p className='mb-2'>Octopus sets out to be the new primary research record: a place where researchers from all disciplines can freely record all of their research outputs without external requirements unduly shaping their outputs.</p><p className='mb-2'>This complements the traditional paper, which continues to be a good place to disseminate research summaries (and particularly &apos;findings&apos;) to a wider audience: publishing in Octopus doesn't stop you publishing in a journal down the line should you want – but it is the place where you establish your priority for all to see, and document your work in full detail in order to do so, rather like a patent office.</p> <p className='mb-2'>Octopus publications are smaller units. These publications are linked together, reflecting different stages of the research process, allowing each piece of work to be done by different researchers at different times where appropriate. They can be published individually, without needing to fit it into the structure of a traditional journal article. To read more about Octopus' publication types, click <a className='underline' href='https://www.octopus.ac/faq#how_octopus' target='_blank'>here.</a></p><p className='mb-2'>You can also publish on Octopus any work you have previous published in a journal if you retained the copyright, or published CC-BY. Doing this allows your work to get higher visibility and can help it be recognised by your institution or funders.</p> <p className='mb-2'>Our aim is that the different structure of Octopus will help researchers approach their work differently and allow people to concentrate on intrinsic excellence rather than chasing &apos;impact&apos;.</p>"
```
Think this is missing a paragraph break - added in suggestion